moveit_ros/moveit_servo/launch/ros_demo_joint_jog.launch.py

```diff
@@ -0,0 +1,120 @@
+import os
```

Comment:
Noting that this file doesn't seem to do anything yet.

I'd recommend pulling `servo_keyboard_input.cpp` out of the tutorials and putting it here instead, so users have something convenient to test with.

https://github.com/ros-planning/moveit2_tutorials/blob/main/doc/examples/realtime_servo/src/servo_keyboard_input.cpp

Reply:
> I'd recommend pulling `servo_keyboard_input.cpp` out of the tutorials and putting it here instead, so users have something convenient to test with.

That sounds like a good idea.
I already have an updated version of the `servo_keyboard_input.cpp`.